log/standardLogger_test.go

```diff
@@ -249,6 +251,52 @@ func TestPutFields(t *testing.T) {
 	}
 }
 
+func TestLogCaller(t *testing.T) {
```

Comment:
Need to add the same test to the mock logger

Reply:
Yes. I largely ignored tests for the proposal.